server/@types/bapv.d.ts

```diff
@@ -184,8 +184,8 @@ export type VisitSessionData = {
     previousRestrictions?: OffenderRestriction[]
   }
   visit?: VisitSlot
+  originalVisitSlot?: VisitSlot
```

Comment:
Removed `previousVisitRestriction` as storing `originalVisitSlot` includes this.

(Side note - just wondering if `visit` here would be better named as `visitSlot` because that's what it actually is, rather than the visit itself! Any thoughts, @psoleckimoj ?)

Reply:
@tpmcgowan sounds good to me 👍

Reply:
Less confusion elsewhere 🙂

Reply:
👍 will rename that then - but as a separate PR